Assets/_Scripts/Gameplay/ReviewDataSO.cs

```diff
@@ -10,6 +10,15 @@ public class ReviewDataSO : ScriptableObject
         public MusicianAppropriatenessFeedback musicianChoiceFeedback;
         public InstrumentFeedback instrumentChoiceFeedback;
         public AffinityFeedback affinityFeedback;
+        public ReviewImageChoices reviewImageChoices;
+        public int stars;
```

Comment:
Note that stars are originally an Enum. Maybe this should be renamed to be "half-stars" to be more descriptive.

Reply:
Changed to half-stars